client/src/components/CreateOrUpdateForm/CreateOrUpdateForm.js

```diff
@@ -656,6 +643,39 @@ export default class CreateOrUpdateForm extends React.Component {
             }
         }
 
+        /*
+        The following form validation is based off of a conversation with Dr. Decker where the following truth table was
+        defined for various form states of known vs unknown dates:
+        _______________________________________________________________________________
+        DateOfCreation     |DateOfUninstall    |UninstallReasonGiven   |FormStateLegal?|
+        ___________________|___________________|_______________________|_______________|
+        Unknown            |Unknown            |NotGiven               |FALSE          |
```

Comment:
The error message for this condition is confusing or could be more specific. Maybe it could be 'Both date of creation and date of un-install cannot be unknown'

Reply:
And maybe a similar message for the condition underneath